src/Compilers/CSharp/Portable/Symbols/Compilation_WellKnownMembers.cs

```diff
@@ -906,6 +913,42 @@ private static bool AddFlags(TypeSymbol type, ArrayBuilder<bool> transformFlagsB
 
                 // Continue walking types
                 return false;
+
+                static void handleFunctionPointerType(FunctionPointerTypeSymbol funcPtr, ArrayBuilder<bool> transformFlagsBuilder, bool isNestedNamedType, bool addCustomModifierFlags)
+                {
+                    Func<TypeSymbol, (ArrayBuilder<bool>, bool), bool, bool> visitor = addFlags;
+
+                    transformFlagsBuilder.Add(false);
+
+                    var sig = funcPtr.Signature;
+                    handle(sig.RefKind, sig.RefCustomModifiers, sig.ReturnTypeWithAnnotations);
+
+                    foreach (var param in sig.Parameters)
+                    {
+                        handle(param.RefKind, param.RefCustomModifiers, param.TypeWithAnnotations);
+                    }
+
+                    void handle(RefKind refKind, ImmutableArray<CustomModifier> customModifiers, TypeWithAnnotations twa)
```

Comment:
>handle [](start = 25, length = 6)

nit: we've tried to avoid nested local function, for readability. If moved up, could rename to `handleFunctionPointerParameter`
nit: `twa` -> `type`, `customModifiers` -> `refCustomModifiers`

Reply:
It cannot, it captures things from the current scope.

---
In reply to: [422327380](https://github.com/dotnet/roslyn/pull/44078#discussion_r422327380) [](ancestors = 422327380)

Reply:
It doesn't have to capture things from the current scope :-)

---
In reply to: [422416731](https://github.com/dotnet/roslyn/pull/44078#discussion_r422416731) [](ancestors = 422416731,422327380)